src/templates/DatasetSearch/DatasetSearch.jsx

```diff
@@ -159,42 +163,45 @@ const DatasetSearch = ({
   const { theme, keyword } = separateFacets(data ? data.data.facets : []);
 
   return (
+    <>
+    <PageHeader headerText={pageTitle} />
     <section className="ds-l-container">
-      <h1 className="dc-search-header ds-title ds-u-margin-y--3">{pageTitle}</h1>
       <div className="ds-l-row">
-        <div className="ds-l-col--12">
-        {introText ? introText : null}
+        <div className="ds-l-col--12 ds-u-margin-bottom--3">
```

Comment:
This works well. It might be better to allow this to be a prop that accepts an array of titles and nodes so it could be a little more flexible in the future.